src/supyo.cc

```diff
@@ -121,8 +126,8 @@ NAN_METHOD(Detect) {
     printf("# width_step %d\n", width_step);
   }
 
-	// perform detection with the pico library
-	t = getticks();
+  // perform detection with the pico library
+  t = getticks();
 
   // a structure that encodes object appearance
   static unsigned char appfinder[] = {
```

Comment:
I don't know why this is here, but it would be best to put it in file scope and `const`-qualified (I just noticed in the c-s I forgot to add it). This array is not mutated in the function, so there's no reason to keep it inside anyway (even if it's not stack-allocated, given the `static` qualifier).